inst/shiny/server.R

```diff
@@ -329,7 +335,8 @@ shinyServer(function(input, output, session) {
       vals$diffexgenelist <- NULL
       vals$gsvaRes <- NULL
       vals$gsvaLimma <- NULL
-      vals$visplotobject <- NULL
+      visplotobject1 <- NULL
```

Comment:
@mmkhan19 should these be `vals$visplotobject`?

Reply:
I had missed the 'vals' part, updated my code. Thanks. In case you are wondering about why two objects were created? because some new updates with plotly were failing and the object didn't help in plotting the heatmap. So, created two objects, visplotobject1 is for barplot, scatter and boxplot(uses plotly) and obj2 is for heatmap.